src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalBuildContext.cs

```diff
@@ -592,23 +592,34 @@ private static DependencyGraph ConstructDependencyGraphFromLast(DependencyGraph
 
         private static IncrementalStatus GetBuildInfoIncrementalStatus(BuildInfo cb, BuildInfo lb)
         {
+            var details = String.Empty;
```

Comment:
>String.Empty [](start = 26, length = 12)

null

Reply:
Solved.

Reply:
how about `string details = null`

---
In reply to: [200141346](https://github.com/dotnet/docfx/pull/3002#discussion_r200141346) [](ancestors = 200141346)

Reply:
This is for the **left-align**.

Reply:
Done